src/platform/plugins/shared/workflows_management/public/widgets/workflow_yaml_editor/ui/decorations/use_focused_step_outline.ts

```diff
@@ -24,32 +24,36 @@ export const useFocusedStepOutline = (editor: monaco.editor.IStandaloneCodeEdito
           backgroundColor: 'rgba(0, 120, 212, 0.02)',
           border: `1px solid #0078d4`, // Explicit blue color
           borderLeft: `1px solid #0078d4`, // Explicit blue color
-          borderRadius: '3px',
+          borderRadius: '4px',
           boxShadow: `0 1px 3px rgba(0, 120, 212, 0.1)`,
           position: 'relative', // Enable relative positioning for action buttons
+          width: 'calc(100% - 24px) !important',
         },
         '.workflow-step-selected-first': {
           backgroundColor: 'rgba(0, 120, 212, 0.02)',
           borderTop: `1px solid #0078d4`, // Explicit blue color
           borderLeft: `1px solid #0078d4`, // Explicit blue color
           borderRight: `1px solid #0078d4`, // Explicit blue color
-          borderTopLeftRadius: '3px',
-          borderTopRightRadius: '3px',
+          borderTopLeftRadius: '4px',
+          borderTopRightRadius: '4px',
           position: 'relative', // Enable relative positioning for action buttons
+          width: 'calc(100% - 24px) !important',
```

Comment:
Let's leave some comment explaining 24px magic number
```suggestion
          width: 'calc(100% - 24px) !important', // To add a space between decoration and the scrollbar  
```